src/index.tsx

```diff
@@ -6,14 +6,26 @@ import { Provider } from 'react-redux';
 import './index.css';
 import App from './components/App';
 import configureStore from './configureStore';
+import { actions as apiActions } from './reducers/api';
 
 const store = configureStore();
 
+const rootElement = document.getElementById('root') as HTMLElement;
+const authToken = rootElement ? rootElement.dataset.authToken : null;
+
+if (
+  authToken &&
+  authToken.length &&
+  authToken !== process.env.REACT_APP_AUTH_TOKEN_PLACEHOLDER
+) {
+  store.dispatch(apiActions.setAuthToken({ authToken }));
```

Comment:
I'd rather see this happen in `App` within `componentDidMount()`. That will make it easier to test.

Reply:
Yep, that works for me but it means: no API call can be made in `componentDidMount()` anymore.